### PR TITLE
add p2 nav and slider; update README

### PR DESCRIPTION
**Pull request by iRohitGaur**
Responsive Nav at 768px
sui-nav-mobile-1
Responsive Nav at 600px
sui-nav-mobile-2
Standard Slider
slider-standard
Slider with Icon
slider-with-icon
Disabled Slider
slider-disabled

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -259,3 +260,19 @@ StarlingUI covers the following types of Grid:
...
+
+StarlingUI covers the following types of Slider:
+
+- Standard Slider
+- Slider with Icon
+- Disabled Slider with Icon
+
+Check it out at: https://starlingui.netlify.app/pages/components/slider/slider.html
+
+---
```
Comment (manojsarna) [inline README.md:276]: Good Work !!

Keeping up with the regular documentation

Suggestion : 

You can have a updation Tab for your component library  for the end users to get the latest updates you have done in your CL.

You can document your CL like that :)

**Review by saurabhan — APPROVED** [review]
UI looks good, clean, and readable code. approved

File: pages/components/navigation/navigation.css [hunk 2]
```diff
@@ -32,8 +35,19 @@
...
+.sui_search_mobile {
+  grid-area: search-mobile;
+  display: none;
+  width: 100%;
+}
+.sui_cta_wrapper {
+  grid-area: cta;
+  margin-left: auto;
+  position: relative;
+}
```
Comment (manojsarna) [inline pages/components/navigation/navigation.css:50]: good naming for your grid areas :)

**Review by zoef-yt — APPROVED** [review]
The UI looks amazing. The code looks amazing. Nothing to suggest from my side. Nice use of comments. Great pr description. Good to merge.

File: pages/components/navigation/navigation.css [hunk 3]
```diff
@@ -43,7 +57,44 @@
...
+  }
+  .sui_search_mobile {
+    display: revert;
+  }
+}
+@media only screen and (max-width: 600px) {
+  .sui_cta_wrapper {
+    flex-direction: column;
+  }
+  .sui_nav_mobile .sui_cta {
```
Comment (manojsarna) [inline pages/components/navigation/navigation.css:84]: Maintaining standard screen sizes for media query is great :)

**Review by varun13169 — APPROVED** [review]

File: pages/components/navigation/navigation.css [hunk 4]
```diff
@@ -43,7 +57,44 @@
...
+    flex-direction: column;
+    position: absolute;
+    background: var(--BG);
+    top: 5rem;
+    right: -1rem;
+    z-index: 9;
+    padding: 2rem;
```
Comment (manojsarna) [inline pages/components/navigation/navigation.css:91]: Why are you using high value of z-index ?

**Comment by varun13169** [comment]
Consistent code with consistent UI, Nice.

File: pages/components/navigation/navigation.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
+const burgerBtn = document.querySelector("#burger");
+const mobileNav = document.querySelector(".collapsed_cta");
+
+burgerBtn.addEventListener("click", () => {
+  mobileNav.classList.contains("hide_cta")
+    ? mobileNav.classList.remove("hide_cta")
+    : mobileNav.classList.add("hide_cta");
+});
```
Comment (manojsarna) [inline pages/components/navigation/navigation.js:8]: easily understandable code. Good Work :)

**Review by manojsarna — APPROVED** [review]
UI is Fantabulous..!!

Great Work Rohit!!

Do Check the comments !!

File: README.md (hunk 1 above)
Comment (iRohitGaur) [reply README.md:276]: Thanks for the suggestion. It is a good idea. Will definitely keep it in the books for future implementation.

File: pages/components/navigation/navigation.css (hunk 4 above)
Comment (iRohitGaur) [reply pages/components/navigation/navigation.css:91]: To override the existing value of my Prism Highlight.

**Pull request merged**